Project.toml

```
@@ -3,12 +3,13 @@ uuid = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
 keywords = ["markov chain monte carlo", "probablistic programming"]
 license = "MIT"
 desc = "A lightweight interface for common MCMC methods."
-version = "4.1.3"
+version = "4.1.4"
```

This should be
```suggestion
version = "4.2"
```
shouldn't it? It's a new feature, not a bugfix.

---

Oh true :+1: